frontend2/src/components/elements/Button.tsx

```diff
@@ -4,6 +4,9 @@ import Icon, { type IconName } from "./Icon";
 const VARIANTS: Record<string, string> = {
   "": "bg-gray-50 text-gray-800 hover:bg-gray-100 hover:ring-gray-900 hover:text-black ring-gray-500 ring-1 ring-inset",
   dark: "bg-cyan-700 text-white hover:bg-cyan-800",
+  danger: "bg-red-700 text-white hover:bg-red-800",
```

Comment:
Nit: should we make a "type variant = 'dark' | 'danger' | ..." so this can be a record from variant to string?

Reply:
This would be useful for type inference for example